utilities/test_plugins/custom_plugins/handler_plugins.py

```diff
@@ -11,3 +12,29 @@ class WrongSubclass(list):
 
 class CustomIOHandler(ParameterIOHandler):
     _FORMAT = "JSON"
+
+class BuckinghamHandler(_NonbondedHandler):
+    """A custom parameter handler for buckingham interactions."""
+
+    class BuckinghamType(ParameterType):
```

Comment:
(blocking) Could these be given names like `TestBuckinghamHandler` and `TestBuckinghamType` to remove all chance that they come into conflict with a real plugin in the future?

Reply:
Good idea - I'm rolling with `FOO` as a prefix, though, just in case it's collected as a test by `pytest`